### PR TITLE
Add instructions and allow to run locally

### PR DESCRIPTION
**Pull request by debugwand**
🐿 v2.10.3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +55,9 @@ If you've used the shell script or if you've checked out, installed and built al
   ```
 - restart `next-router`
 - `cd` into `next-syndication-api` and `make run-local`
+- go to https://local.ft.com:5050/syndication/user-status to confirm everything is working
```
Comment (leggsimon) [inline README.md]:
```suggestion
- go to [https://local.ft.com:5050/syndication/user-status](https://local.ft.com:5050/syndication/user-status) to confirm everything is working
```

File: README.md [hunk 2]
```diff
@@ -49,6 +55,9 @@ If you've used the shell script or if you've checked out, installed and built al
...
+
+If you see an error that port 3255 is already being used, use `lsof -n -i :3255 | grep LISTEN` to find out its PID and `ps -p THE_PID` to find out what that process is. Sometimes PM2 doesn't stop properly, and you can use `kill THE_PID` to stop it.
```
Comment (leggsimon) [inline README.md]: I wonder if we need this step? Is this going to be a common thing?

Comment (debugwand) [reply README.md]: the 3255 port error? maybe not, that was because I'd been running make run-local and make run.

Comment (leggsimon) [reply README.md]: Maybe see what Alice comes up against?

**Pull request merged**